pkg/executor/bacalhau/bacalhau.go

```diff
@@ -38,6 +38,30 @@ func NewBacalhauExecutor(options BacalhauExecutorOptions) (*BacalhauExecutor, er
 	}, nil
 }
 
+func (executor *BacalhauExecutor) Id() (string, error) {
+	nodeIdCmd := exec.Command(
+		"bacalhau",
+		"id",
+	)
+	nodeIdCmd.Env = executor.bacalhauEnv
+
+	output, err := nodeIdCmd.CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("error calling get id results %s", err.Error())
+	}
+
+	var idResult struct {
+		ID       string
+		ClientID string
+	}
+	err = json.Unmarshal(output, &idResult)
+	if err != nil {
+		return "", fmt.Errorf("error unmarshalling job JSON %s", err.Error())
+	}
+
+	return idResult.ID, nil
```

Comment:
it's not a issue, I'm just curious what does the ID look like/what is it? some random uuid, the server name, ...?

Reply:
a peer id

Reply:
eg QmS93KCp7HQLQexRwcxpvAFvTLpDdkNPVUsSPgZtjDSLpT